src/main/java/com/abdullah/coding/challenge/controllers/CustomerController.java

```diff
@@ -0,0 +1,30 @@
+package com.abdullah.coding.challenge.controllers;
+
+import com.abdullah.coding.challenge.Response;
+import com.abdullah.coding.challenge.services.BookingService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/customer")
+public class CustomerController {
```

Comment:
CustomerController here does only caters to specific customer related functions. I deemed it more appropriate to use BookingController. It can be argued that it is redundant or that the booking controller has too many responsibilities.